cmd/operator/main.go

```diff
@@ -91,31 +99,33 @@ func init() {
 }
 
 func main() {
+	// Logger.
 	logger := logrus.New()
 	loggingLevel, err := logrus.ParseLevel(flLoggingLevel)
 	if err != nil {
 		logger.Fatalf("invalid logging level: %v", err)
 	}
 	logger.SetLevel(loggingLevel)
 
+	if !isatty.IsTerminal(os.Stdout.Fd()) {
+		logger.Formatter = stackdriver.NewFormatter(
+			stackdriver.WithService("cloud-run-release-operator"),
```

Comment:
in the future we can use os.Getenv(K_SERVICE) as a first attempt to find the value